visualization/app/codeCharta/ui/codeMap/rendering/boxGeometryGenerationHelper.ts

```diff
@@ -164,31 +177,35 @@ export class BoxGeometryGenerationHelper {
 				uvs[intermediateIndexBL],
 				color,
 				subGeomIndex,
-				deltaRelativeToHeight
+				deltaRelativeToHeight,
+				isTopSide[0]
```

Comment:
Does it make sense to use the enum from line 6 for the side index?
`sides.left` in this case.

Reply:
I'm unsure if the enum would match the numbering, I will further investigate it

Reply:
I think it wouldn't make much sense. Imagine: `isTopSide(sides.Left)`
So it's good to keep the current implementation.